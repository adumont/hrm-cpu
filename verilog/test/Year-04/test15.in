56 98
57 5c
59 b7
29 cf
41 73
09 64
37 d3
13 2c
51 d1
42 61
11 0a
53 30
28 34
13 ad
48 97
54 45
20 2c
17 65
29 ff
18 0d
16 94
25 81
26 d4
18 78
43 55
58 37
50 de
44 25
04 4d
41 21
52 3c
57 36
33 75
32 79
43 70
