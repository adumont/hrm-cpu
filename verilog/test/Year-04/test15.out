5c
98
cf
b7
64
73
2c
d3
61
d1
30
0a
ad
34
45
97
65
2c
0d
ff
81
94
78
d4
37
55
25
de
21
4d
36
3c
79
75
